usafacts/delphi_usafacts/geo.py

```diff
@@ -92,43 +92,41 @@ def geo_map(df: pd.DataFrame, geo_res: str, sensor: str):
     # Disburse unallocated cases/deaths in NYC to NYC counties
```

Comment:
Just a final nitpick to make those masks identical lol.

Reply:
:+1: